File: app/src/main/java/com/example/englishquiz/Question.java
package com.example.englishquiz;

public class Question {
    public int questionResId;
    public int firstOptionResId;
    public int secondOptionResId;
    public int explanationResId;
    public int rightAnswer;

    public Question(int questionResId,
                    int firstOptionResId,
                    int secondOptionResId,
                    int explanationResId,
                    int rightAnswer) {
        this.questionResId = questionResId;
        this.firstOptionResId = firstOptionResId;
        this.secondOptionResId = secondOptionResId;
        this.explanationResId = explanationResId;
        this.rightAnswer = rightAnswer;
    }
}

File: app/src/main/java/com/example/englishquiz/Question.kt
package com.example.englishquiz

import androidx.annotation.StringRes

data class Question(
    @StringRes val questionResId: Int,
    @StringRes val firstOptionResId: Int,
    @StringRes val secondOptionResId: Int,
    @StringRes val explanationResId: Int,
    val rightAnswerResId: Int
)